include/clspv/Option.h

```diff
@@ -172,7 +172,8 @@ SourceLanguage Language();
 // Returns true when the source language makes use of the generic address space.
 inline bool LanguageUsesGenericAddressSpace() {
   return (Language() == SourceLanguage::OpenCL_CPP) ||
-         ((Language() == SourceLanguage::OpenCL_C_20));
+         (Language() == SourceLanguage::OpenCL_C_20) ||
+         (Language() == SourceLanguage::OpenCL_C_30);
```

Comment:
It feels like this should be based on the feature being enabled. I'm not sure by default is the right choice given it requires full inlining. If clvk wants to always enable it that's ok, but I'm a bit wary of forcing that decision on all users.